diff --git a/node_modules/cytoscape-elk/dist/cytoscape-elk.js b/node_modules/cytoscape-elk/dist/cytoscape-elk.js
index e64b4d1..f523c5a 100644
--- a/node_modules/cytoscape-elk/dist/cytoscape-elk.js
+++ b/node_modules/cytoscape-elk/dist/cytoscape-elk.js
@@ -179,7 +179,8 @@ var getPos = function getPos(ele, options) {
 var makeNode = function makeNode(node, options) {
   var k = {
     _cyEle: node,
-    id: node.id()
+    id: node.id(),
+    layoutOptions: node.data('layoutOptions')
   };
 
   if (!node.isParent()) {
@@ -205,6 +206,10 @@ var makeEdge = function makeEdge(edge
     source: edge.data('source'),
     target: edge.data('target')
   };
+  const label = edge.style("label");
+  if (label) {
+    k['labels'] = [{text: label}];
+  }
   edge.scratch('elk', k);
   return k;
 };
@@ -293,9 +298,8 @@ var Layout = /*#__PURE__*/function () {
       var edges = eles.edges();
       var elk = new (external_commonjs_elkjs_commonjs2_elkjs_amd_elkjs_root_ELK_default())();
       var graph = makeGraph(nodes, edges, options);
-      elk.layout(graph, {
-        layoutOptions: options.elk
-      }).then(function () {
+      graph['layoutOptions'] = options.elk;
+      elk.layout(graph).then(function () {
         nodes.filter(function (n) {
           return !n.isParent();
         }).layoutPositions(layout, options, function (n) {
